GenderPayGap.WebUI/Models/Admin/DatabaseIntegrityChecksViewModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
```

Comment:
This file might not be needed - it might work if the model for each ajax call is just `List<Organisation>`

Reply:
Ahh, I've only just seen this after writing my last comment. Thanks again